tests/fixture/database/test_patch.py

```diff
@@ -17,35 +13,29 @@
 redshift = create_redshift_fixture()
 
 
-@patch_create_engine(path="tests.fixture.database.create_engine")
 def test_copy(redshift):
     copy_fn_to_test_create_engine_patch(redshift)
 
 
-@patch_create_engine(path="tests.fixture.database.create_engine")
 def test_unload(redshift):
     unload_fn_to_test_create_engine_patch(redshift)
 
 
-@patch_psycopg2_connect(path="tests.fixture.database.psycopg2")
 def test_copy_with_psycopg2(redshift):
     config = redshift.pmr_credentials.as_psycopg2_kwargs()
     copy_fn_to_test_psycopg2_connect_patch(config)
 
 
-@patch_psycopg2_connect(path="tests.fixture.database.psycopg2")
 def test_copy_with_psycopg2_as_context_manager(redshift):
     config = redshift.pmr_credentials.as_psycopg2_kwargs()
     copy_fn_to_test_psycopg2_connect_patch_as_context_manager(config)
 
 
-@patch_psycopg2_connect(path="tests.fixture.database.psycopg2")
 def test_unload_with_psycopg2(redshift):
     config = redshift.pmr_credentials.as_psycopg2_kwargs()
     unload_fn_to_test_psycopg2_connect_patch(config)
 
 
-@patch_psycopg2_connect(path="tests.fixture.database.psycopg2")
```

Comment:
these should probably still be here as well, surprised the tests are passing

Reply:
See src/pytest_mock_resources/patch/redshift/psycopg2.py, different impl. Which I'm now recalling that was required because of the change to config. the old version directly relied on global imports of the config, so it needed to be adjusted.

In fact, I assume this is why I removed the documentation for `patch_create_engine`, because it's no longer required, due to the new mechanism that patches out the pyscopg2 connection directly, rather than at the sqlalchemy level.

Reply:
nice 👍